CONTRIBUTING.md

```diff
@@ -1,33 +1,37 @@
 
 ## Contribution Process
-1. Fork the project, and make changes relevant to what you're trying to add/fix on a seperate branch
-2. Create a Github Pull Request for your change in your specific branch. You should include a description of the change, why you're requesting your change to be made, and where you tested your change
-3. Don't expect your changes to be merge to upstream, since a team member will review your pull request, asking for further possible changes
-4. Once all further changes have been met (if any) and your changes are approved, it will be merged to upstream and the APT repository shortly updated
+1. Fork the project and make changes relevant to what you're trying to add/fix — do this in a seperate branch in your fork
+2. Test your changes made on a **physical** device. If you're adding or updating a package, test your changes with a build of the package you added/updated
+3. Create a Pull Request for your changes in your specific branch — include a description of the change and where you tested your change
+
+Don't expect your changes to be merged to pushed to upstream immediately — your changes will be reviewed by a team member (or Hayden), providing suggested changes which must be met.
+
+Once all further changes have been met (if any) and your changes are approved, they will be merged into upstream. The APT repository will shortly update showcasing your changes.
 
 ### Adding a New Package
-Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files, only: a ``.mk`` file and a ``.control`` file. For documented examples, take a look at the [``grep.mk`` template](./grep.mk.template) and its [control file](./grep.control).
+Adding new packages to Procursus is fairly easy and simple. For most packages, you're likely to need 2 files
+- A ``.mk`` file, that builds the package, and..
+- A ``.control`` file, which contains info about your package
 
 Some things to keep in mind when adding a new package
-- Always ensure that the Architecture and Maintainer fields of your control file are populated with ``@DEB_ARCH@`` and ``@DEB_MAINTAINER@``, respectively
-- If you take advantage of already-made patchfiles from an external source, download them and implement them to your project. Check out it's done in [``bash.mk``](./bash.mk)
-- If the package you're adding only requires a few small edits, try taking advantage of ``sed`` in your packages' setup stage
+- Always ensure that the Architecture and Maintainer fields of your control file are populated with ``@DEB_ARCH`` and ``@DEB_MAINTAINER@``, respectively
+- If you take advantage of already-made patchfiles from an external source, download them and implement them to your project. Check out how it's done in [``bash.mk``](./makefiles/bash.mk)
+- If the package you're adding only requires a few small edits, try taking advantage of ``sed`` in your packages' setup stage, and..
```

Comment:
```suggestion
- If the package you're adding only requires a few small edits, try taking advantage of ``sed`` in your package's setup stage
```

Reply:
"Packages's" translates to "package is"; the correct way is "packages'", since it's an attribute the package has, not what the package is.

Reply:
Are you dumb? If you're referring to a single package, it's `package's`. `packages'` or `packages's` would be referring to *multiple* packages.

Reply:
My bad; I'll unresolve and agree with the changes.